app/views/users/followers.html.erb

```diff
@@ -0,0 +1,37 @@
+<!-- app/views/users/followers.html.erb -->
+
+<div class="row">
+  <div class="col md-6 offset-3">
```

Comment:
i believe you are supposed to add a '-' between col and md-6 to make it all align perfectly. so it should be class="col-md-6 offset 3"